### PR TITLE
Disable episodes in list when episode hasn't aired yet

### PR DESCRIPTION
**Pull request by JipFr**
This pull request resolves #XXX

 - [x] I have read and agreed to the [code of conduct](https://github.com/movie-web/movie-web/blob/dev/.github/CODE_OF_CONDUCT.md).
 - [x] I have read and complied with the [contributing guidelines](https://github.com/movie-web/movie-web/blob/dev/.github/CONTRIBUTING.md).
 - [x] What I'm implementing was assigned to me and is an [approved issue](https://github.com/movie-web/movie-web/issues?q=is%3Aopen+is%3Aissue+label%3Aapproved). For reference, please take a look at our [GitHub projects](https://github.com/movie-web/movie-web/projects).
 - [x] I have tested all of my changes.

image

### CODE REVIEW

**Comment by zisra** [comment]
A lot of unaired episodes are available with certain providers though

**Comment by JipFr** [comment]
Such as?

**Comment by zisra** [comment]
The mandalorian (they're all aired now) but it worked while Disney+ was still releasing episodes

**Comment by JipFr** [comment]
Providers already had episodes when they hadn't come out on Disney+? How?

File: src/assets/locales/en.json [hunk 1]
```diff
@@ -189,7 +189,8 @@
         "loadingList": "Loading...",
         "loadingError": "Error loading season",
         "emptyState": "There are no episodes in this season, check back later!",
-        "episodeBadge": "E{{episode}}"
+        "episodeBadge": "E{{episode}}",
+        "unairedEpisodes": "One or more episodes in this season have been disabled because they haven't aired yet."
```
Comment (mrjvs) [inline src/assets/locales/en.json:193]: `One or more episodes in this season have been disabled, as they haven't been aired yet.`
How about this?

**Review by mrjvs — CHANGES_REQUESTED** [review]

**Review by mrjvs — APPROVED** [review]

**Pull request merged**